<ui version="4.0" >
 <class>DTWindow</class>
 <widget class="QWidget" name="DTWindow" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Detectando Transições</string>
  </property>
  <property name="windowIcon" >
   <iconset>
    <normaloff>Icons/logo.png</normaloff>Icons/logo.png</iconset>
  </property>
  <widget class="QLabel" name="meesageLabel" >
   <property name="geometry" >
    <rect>
     <x>140</x>
     <y>10</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Detectando Transições</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>30</y>
     <width>353</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value" >
    <number>24</number>
   </property>
  </widget>
  <widget class="QFrame" name="frame_3" >
   <property name="geometry" >
    <rect>
     <x>38</x>
     <y>60</y>
     <width>301</width>
     <height>100</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout" >
    <item row="0" column="0" >
     <widget class="QLabel" name="label_3" >
      <property name="frameShape" >
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth" >
       <number>0</number>
      </property>
      <property name="text" >
       <string>Tipo de Transição</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" >
     <widget class="QLabel" name="labelType" >
      <property name="frameShape" >
       <enum>QFrame::Box</enum>
      </property>
      <property name="text" >
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2" >
     <widget class="Line" name="line" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0" >
     <widget class="QLabel" name="label_5" >
      <property name="frameShape" >
       <enum>QFrame::Box</enum>
      </property>
      <property name="lineWidth" >
       <number>0</number>
      </property>
      <property name="text" >
       <string>Etapa da Detecção</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" >
     <widget class="QLabel" name="labelFase" >
      <property name="frameShape" >
       <enum>QFrame::Box</enum>
      </property>
      <property name="text" >
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2" >
     <widget class="Line" name="line_2" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="0" >
     <widget class="QLabel" name="label_4" >
      <property name="frameShape" >
       <enum>QFrame::Box</enum>
      </property>
      <property name="lineWidth" >
       <number>0</number>
      </property>
      <property name="text" >
       <string>Total de Transições</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" >
     <widget class="QLabel" name="labelNum" >
      <property name="frameShape" >
       <enum>QFrame::Box</enum>
      </property>
      <property name="text" >
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
